src/components/Pagination/PaginationService.js

```diff
@@ -0,0 +1,79 @@
+import {
+  ceil,
+  compose,
+  divide,
+  isEmpty,
+  first,
+  last,
+  gt,
+  add,
+  subtract,
+  lt,
+  __,
+  sortBy,
+  identity,
+  reverse
+} from 'lodash/fp';
+
+export const calculatePages = (totalItems, perPage) =>
+  compose(ceil, divide(totalItems))(perPage);
+
+export const createEmptyArrayFromNumber = length => Array.from({ length });
```

Comment:
is this abstraction necessary? `Array.from({ length })` seems pretty clear to me